movie-app/src/redux/rootReducer.ts

```diff
@@ -14,45 +15,45 @@ export const initialState: any = {
   searchParams: {
     sortBy: 'release_date',
     searchBy: 'title',
-    search: '',
+    search: ''
   },
-  selectedMovie: null,
+  selectedMovie: null
 };
 
 const reduxThunkReducer = (state = initialState, action: any) => {
   switch (action.type) {
     case LOAD_MOVIES_LOADING: {
-      return {
+      return Map({
```

Comment:
I'm not sure this is a proper usage of immutable. https://medium.com/front-end-weekly/using-immutable-js-with-redux-ba89025e45e2